thirdparty/setuptools-41.2.0-py2.py3-none-any.whl.ABOUT

```diff
@@ -0,0 +1,15 @@
+about_resource: setuptools-41.2.0-py2.py3-none-any.whl
```

Comment:
We should already have setuptools..... good catch. But please do not commit it there, craft another PR instead just for that.

Reply:
Don't worry, i already mentioned in PR description.

Reply:
I get that, but please provide a different PR for adding that back (and FYI https://github.com/zopefoundation/zc.lockfile/issues/22 )

Reply:
removed and PR https://github.com/nexB/scancode-toolkit/pull/2123